### PR TITLE
Skatepark: Add default duotone configuration to theme.json

### LINKED ISSUE #4329: Skatepark: Duotone theme-level configuration
Skatepark includes in its design the application of duotone filter on the images it provides. The theme should apply this filter to all relevant blocks by default.

This will require https://github.com/WordPress/gutenberg/issues/33642 to be addressed, which will allow us to set a filter as default (and we should consider changing the default filter when the user changes the theme color as well).

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
This adds a default duotone filter to all image blocks.

Needs https://github.com/WordPress/gutenberg/pull/34073/files

#### Related issue(s):
Fixes https://github.com/Automattic/themes/issues/4329

### CODE REVIEW

File: skatepark/child-theme.json [hunk 1]
```diff
@@ -310,6 +310,16 @@
 				"typography": {
 					"letterSpacing": "0.05em"
 				}
+			},
+			"core/image": {
+				"color": {
+					"duotone": "var(--wp--preset--duotone--red-filter)"
+				}
+			},
+			"core/cover": {
```
Comment (scruffian) [inline skatepark/child-theme.json]: This one doesn't work yet.

File: skatepark/theme.json [hunk 2]
```diff
@@ -537,6 +537,11 @@
 					"fontStyle": "normal"
 				}
 			},
+			"core/post-author": {
+				"color": {
+					"duotone": "url(#wp-duotone-red-filter)"
+				}
+			},
```
Comment (MaggieCabrera) [inline skatepark/theme.json]: I added this too

Comment (MaggieCabrera) [reply skatepark/theme.json]: It works fine!
Screenshot 2021-09-02 at 15 58 23

**Comment by MaggieCabrera** [comment]
I think we can close this one

**Pull request closed without merging**